### PR TITLE
Avtar & badges

### PR DESCRIPTION
**Pull request by jeetbhalani15**
image

### CODE REVIEW

File: components/Badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+@import "/reset.css";
+.box{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```
Comment (rohanmathur91) [inline components/Badges/badges.css]: You can have a utility class for this.

File: components/Badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    margin-top: 80px;
+    justify-content: center;
+    gap: 60px;
```
Comment (rohanmathur91) [inline components/Badges/badges.css]: Use rem instead of px.

File: components/Badges/badges.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 25px;
+}
+.text__badge{
+    position: relative;
+    width: 160px;
+    height: 42px;
```
Comment (rohanmathur91) [inline components/Badges/badges.css]: Use rem, same as above.

**Review by rohanmathur91 — CHANGES_REQUESTED** [review]
Have a utility class for flex and use rem instead of px.

**Review by maitrakhatri — APPROVED** [review]

File: components/Badges/badges.css (hunk 2 above)
Comment (maitrakhatri) [reply components/Badges/badges.css]: > Use rem instead of px.

+1 rest looks good

File: components/Badges/badges.css (hunk 3 above)
Comment (jeetbhalani15) [reply components/Badges/badges.css]: ok done

File: components/Badges/badges.css (hunk 1 above)
Comment (jeetbhalani15) [reply components/Badges/badges.css]: ok done

**Comment by Jay-Gadhiya** [comment]
use rem instead of pixels. everything looks good

File: components/Alert/alert.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="alert-box alert-danger-outline"><b>👿 Danger!</b> This is A Danger alert Space.    </div>
+            <div class="alert-box alert-success-outline"><b>✅ Success!</b> This is A Success alert Space.    </div>
+            <div class="alert-box alert-warning-outline"><b>⚠️ Warning!</b> This is A Waring alert Space.   </div>
+    </div>
+    <!-- 
+        alert with button
+     -->
+     <div class="alert-btn">
+            <div class="alert-box alert-warning"><b>⚠️ Warning!</b> This is A Waring alert Space.</div>
+
```
Comment (Jay-Gadhiya) [inline components/Alert/alert.html:36]: you can erase space or you can use prettier to format code.

File: components/Badges/badges.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
+@import "/reset.css";
+.box{
```
Comment (Jay-Gadhiya) [inline components/Badges/badges.css:2]: give space after the class name to make code batter readable.

**Review by Jay-Gadhiya — APPROVED** [review]

File: components/Badges/badges.css (hunk 5 above)
Comment (jeetbhalani15) [reply components/Badges/badges.css:2]: ok sure

File: components/Alert/alert.html (hunk 4 above)
Comment (jeetbhalani15) [reply components/Alert/alert.html:36]: ok try to use it from now on

**Pull request merged**